_zh-cn/tour/singleton-objects.md

```diff
@@ -13,3 +13,100 @@ language: zh-cn
 next-page: regular-expression-patterns
 previous-page: pattern-matching
 ---
+
+单例对象是一种特殊的类, 有且只有一个实例. 和惰性变量一样, 单例对象是延迟创建的, 当它被引用到的时候创建.
```

Comment:
Chinese punctuation should be used, I suppose

Reply:
Thanks, declan94, I will use Chinese punctuation